ui/components/component-library/base-avatar/base-avatar.scss

```diff
@@ -28,4 +28,5 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  text-transform: uppercase;
```

Comment:
@garrettbear mentioned that we should be changing the text to uppercase via CSS. It's more performant so we can remove changing via javascript @NidhiKJha